src/main/java/com/vladmihalcea/sql/exception/SQLDeleteCountMismatchAssertionError.java

```diff
@@ -6,9 +6,9 @@
  *
  * @author Vlad Mihalcea
  */
-public class SQLDeleteCountMismatchException extends SQLStatementCountMismatchException {
+public class SQLDeleteCountMismatchAssertionError extends SQLStatementCountMismatchAssertionError {
```

Comment:
Renaming the Exception will break backward compatibility. I'm not sure if it's worth.